include/irr/video/IGPUMeshBuffer.h

```diff
@@ -20,6 +20,10 @@ namespace video
 {
 	class IGPUMeshBuffer final : public asset::IMeshBuffer<IGPUBuffer,IGPUDescriptorSet,IGPURenderpassIndependentPipeline>
```

Comment:
doesn't skinned meshbuffer inherit from this?

Reply:
theres no gpu skinned meshbuffer

Reply:
you're correct.